prqlc/prqlc/tests/integration/sql.rs

```diff
@@ -1155,6 +1155,8 @@ fn test_in_values_01() {
     from employees
     filter (title | in ["Sales Manager", "Sales Support Agent"])
     filter (employee_id | in [1, 2, 5])
+    filter (f"{emp_group}.{role}" | in ["sales_ne.mgr", "sales_mw.mgr"])
+    filter (s"{metadata} ->> '$.location'" | in ["Northeast", "Midwest"])
```

Comment:
```suggestion
    filter (s"{metadata} -> '$.location'" | in ["Northeast", "Midwest"])
```
?

Reply:
Ah, good point. 

I thought the tests had passed, but actually the `cargo fmt` test failed before these had a chance to run. So when we fix that we'll get to test this properly.

It will need a run of `task test-rust` to update the snapshots (or `cargo insta test --accept --features=default,test-dbs`)

Reply:
@PrettyWood I think `->>` is the right operator here, since `in ["", ...]` is comparing to a VARCHAR which is what `->>` returns. Reference: https://duckdb.org/docs/extensions/json.html#json-extraction-functions

Reply:
Don't need to delay merging, but FYI adding this to a query in this path https://github.com/PRQL/prql/blob/76c4f767c4312976dba0fbadfa6c27b84e31faa9/prqlc/prqlc/tests/integration/queries/cast.prql would test on duckdb & postgres DBs in CI